peerconnection.go

```diff
@@ -2246,3 +2246,9 @@ func (pc *PeerConnection) setGatherCompleteHandler(handler func()) {
 func (pc *PeerConnection) SCTP() *SCTPTransport {
 	return pc.sctpTransport
 }
+
+// DTLS returns the currently-configured *DTLSTransport or nil
+// if one has not yet been configured
+func (pc *PeerConnection) DTLS() *DTLSTransport {
```

Comment:
I think you can already get this via `DataChannel` or `Transceiver` let me check!

Does this exist in the webrtc-pc API?

Reply:
* https://pkg.go.dev/github.com/pion/webrtc/v3#RTPReceiver.Transport
* https://pkg.go.dev/github.com/pion/webrtc/v3#RTPSender.Transport
* https://pkg.go.dev/github.com/pion/webrtc/v3#SCTPTransport.Transport (The PeerConnection has a global SCTPTransport)